cypress/integration/token-list/index.spec.ts

```diff
@@ -8,4 +17,43 @@ describe('Token list filter', () => {
     cy.get('#background-colors').should('exist');
     cy.get('#border-colors').should('not.exist');
   });
+
+  describe('Visual regression tests', () => {
+    /* TODO:
+      - full page check or multiple section/region checks
+      - need to add check for branch name/batch id for CI runs
+    */
+    it('Basic VRT', () => {
+      // ℹ️ We are able to reference `this.title` because we have bound this describe block to the Cypress context.
+      const testSuiteName = this.title;
+
+      cy.log('[Applitools]: checking if eyes are enabled');
+      if (eyesAreEnabled()) {
+        cy.log('[Applitools]: Eyes are enabled, proceed with eyes check.');
+        const openParams: Partial<Eyes.Open.Options> = {
+          ...DEFAULT_OPEN_PARAMS,
+          testName: testSuiteName,
+          batchName: getBatchName(),
```

Comment:
TODO: This needs (may need) to be updated when you figure out the requirements from Applitools for the batch name and github workflow integration.